bot/cogs/utils.py

```diff
@@ -59,6 +67,19 @@ def __init__(self, bot: Bot):
             await ctx.invoke(self.bot.get_command("help"), "pep")
             return
 
+        # Handle PEP 0 directly due it's not available like other PEPs (use constants)
```

Comment:
This explanation could be a lot better. What is different for PEP 0 vs. the other PEPs that makes the existing logic unusable?